packages/replay-next/pages/variables.css

```diff
@@ -709,7 +716,7 @@
   --theme-text-field-bgcolor: #f6f6f6;
   --theme-text-field-color: var(--body-color);
   --theme-text-field-placeholder-color: #a9a9b1;
-  --team-row-active: #3f434a;
+  --team-row-active: #163e58; /* the dark version of --primary-accent-dimmed */
```

Comment:
btw
```suggestion
  --team-row-active: var(--primary-accent-dimmed);
```

Reply:
Usually variables are great. But in this case it's the left-hand column in the library, which always stays dark. Using the variable here means it uses the wrong color when in light theme.

Reply:
Oh I see. Hmm 🤔 That's pretty easy to get mixed up.

Reply:
Indeed. I run against this every. single. time.